dev_guide/topics/about_device_manager.adoc

```diff
@@ -1,20 +1,5 @@
 == What Device Manager Does
 
-[IMPORTANT]
-====
-Device Manager is a Technology Preview feature.
```

Comment:
@vikaschoudhary16 Can you please confirm that Device Manager is coming out of Tech Preview?

Reply:
Confirmed with Jeremy that it's GA in 3.10